src/ast_403.ml

```diff
@@ -2070,16 +2070,18 @@ module Ast_mapper : sig
 
   val map_opt: ('a -> 'b) -> 'a option -> 'b option
 
-  val extension_of_error: Location.error -> extension
+  val extension_of_error: Locations.location_error -> extension
   (** Encode an error into an 'ocaml.error' extension node which can be
       inserted in a generated Parsetree.  The compiler will be
       responsible for reporting the error. *)
 
-  val attribute_of_warning: Location.t -> string -> attribute
+ val attribute_of_warning: Location.t -> string -> attribute
```

Comment:
this line changed for no reason i believe

Reply:
Indeed.